client/me/help/help-search/index.jsx

```diff
@@ -84,8 +91,8 @@ export class HelpSearch extends React.PureComponent {
 		}
 
 		const forumBaseUrl = helpLinks.wordpress_forum_links_localized
-			? localizeUrl( 'https://en.forums.wordpress.com/' )
-			: 'https://en.forums.wordpress.com/';
+			? localizeUrl( 'https://wordpress.com/forums' )
```

Comment:
Will `localizeUrl` give the same results with these updated URLs?

Reply:
Yep yep!

<img width="994" alt="Screen Shot 2021-01-26 at 12 10 48 PM" src="https://user-images.githubusercontent.com/2124984/105878974-a5d4fc80-5fcf-11eb-9309-0f69ebd6d467.png">